### ISSUE TITLE
Kodi 17 crashes on exit

### ISSUE THREAD

**Issue by axlt2002** [kodi-pvr/pvr.argustv#59]
Hi All,
Just to notify that when Kodi 17 exits and the ArgusTV Client is activated, there is a crash. I have activated the bebug (please refer to the attachment for the complete log). It seems that there are some issues while releasing the add-on.
Hope you can provide fix. Thanks for your effort in advance.

[kodi_bug.log.txt](https://github.com/kodi-pvr/pvr.argustv/files/272737/kodi_bug.log.txt)

**Comment by axlt2002** [comment]
Hi All,
Is there any new or feedback on this issue?
Just to understand if it something related to the client itself or to how Kodi 17 manages the release of the components while exiting.
Thanks a lot for your effort.

**Comment by MartijnKaijser** [comment]
@ksooo @hudokkow you know maintains this addon? It causes crashes and if there's no maintainer i'm gone remove it from our default install

**Comment by axlt2002** [comment]
Thanks MartijnKaijser for your intervention :)
Personally I saw some activities in the development a couple of months ago to change the API version (if I'm not wrong). Afterwards, nothing more...
Did you understand something from the debug log I posted? Personally not... :(

**Comment by hudokkow** [comment]
Quickly browsing the log, I can't see anything wrong with the add-on. It starts, loads channels and then we get
> 15:14:09 T:5352   FATAL: /libcurl.dll is a system dll and should never be released
15:14:09 T:5352   FATAL: /cpluff.dll is a system dll and should never be released
15:14:09 T:5352   FATAL: /libcec.dll is a system dll and should never be released
15:14:09 T:5352   FATAL: C:\Program Files\Kodi\addons\pvr.argustv\pvr.argustv.dll is a system dll and should never be released

ping @margro

**Comment by axlt2002** [comment]
I would in fact agree (that it was the purpose of my question). The add-on currently works perfectly inside Kodi. The issue is just related at the closure of Kodi...it seems that is not possible to release an ArgusTV's dll. Somethig that can be mangaed on the Kodi's code?

**Comment by hudokkow** [comment]
Yeah, sorry about that. Much too soon here and I was in need of a gallon of coffee. Brain is now on.

@Paxxi, any idea about what changed, if anything changed at all with the DLLs? My win experience is 0.
